test/integration/noderesourcetopology_test.go

```diff
@@ -932,7 +931,7 @@ func getNodeName(ctx context.Context, c clientset.Interface, podNamespace, podNa
 func createNodeResourceTopologies(ctx context.Context, topologyClient *versioned.Clientset, noderesourcetopologies []*topologyv1alpha1.NodeResourceTopology) error {
 	for _, nrt := range noderesourcetopologies {
 		_, err := topologyClient.TopologyV1alpha1().NodeResourceTopologies().Create(ctx, nrt, metav1.CreateOptions{})
-		if err != nil && !errors.IsAlreadyExists(err) {
+		if err != nil {
```

Comment:
good catch!

Reply:
This was not a bug. 
It's just wrong.
There is no reason to have the NRT object exist before the test start.
The environment should always stays clean and if it's not we should report it back.

Reply:
sure, agree with your assessment